webapp/channels/src/components/widgets/tag/tag.tsx

```diff
@@ -26,10 +26,6 @@ type Props = {
 type TagWrapperProps = Required<Pick<Props, 'uppercase'>>;
 
 const TagWrapper = styled.div<TagWrapperProps>`
-    --tag-bg: var(--semantic-color-general);
```

Comment:
Why change all of these? The existing version of this code looks a bit odd, but I don't think we should be changing it right now

Reply:
Agreed its not related, but the end code looks a lot more readable.